tests/e2e/specs/product-editor/block-integration.test.js

```diff
@@ -0,0 +1,693 @@
+/**
+ * External dependencies
+ */
+import { expect, test, Page } from '@playwright/test';
+
+/**
+ * Internal dependencies
+ */
+import * as api from '../../utils/api';
+import { getProductBlockEditorUtils } from '../../utils/product-editor';
+
+test.use( { storageState: process.env.ADMINSTATE } );
+test.describe.configure( { mode: 'serial' } );
+
+test.describe( 'Product Block Editor integration', () => {
+	/**
+	 * @type {Page}
+	 */
+	let page = null;
+	let editorUtils = null;
+
+	test.beforeAll( async ( { browser } ) => {
+		page = await browser.newPage();
+		editorUtils = getProductBlockEditorUtils( page );
+
+		await editorUtils.toggleBlockFeature( true );
+		await api.setOnboardedMerchant();
```

Comment:
Apart from the "Prompt to Get Started" case, all other cases need to be tested in the onboarded state.

If it isn't set here, then manual setup will be required when developing other cases, or the "Prompt to Get Started" case will have to be run first.

The main goal is to get the test cases as close to atomicity as possible so that test cases can be tested individually. This makes development and debugging easier and reduces false positives caused by side effects.